soc/silabs/silabs_siwx917/siwg917/Kconfig.soc

```diff
@@ -19,3 +19,34 @@ config SOC
 
 config SOC_PART_NUMBER
 	default "SIWG917M111MGTBA" if SOC_PART_NUMBER_SIWG917M111MGTBA
+
+config SIWX917_NWP_NETWORK_STACK
+	bool
+	select CMSIS_RTOS_V2
+	select POLL
+	select DYNAMIC_THREAD
+	select THREAD_NAME
+	select THREAD_STACK_INFO
+	select THREAD_MONITOR
+	select INIT_STACKS
```

Comment:
I think the reason you're getting Compliance check failures is that you shouldn't have this kind of cross-references from HWMv2 Kconfig files to other Kconfig symbols. I suspect the correct place would be `soc/silabs/silabs_siwx917/siwg917/Kconfig`.

Reply:
I see, makes sense, however I'm not sure where this can be moved.
Do you have any suggestions?

The NWP overhead can be needed by many drivers, like BT, WiFi, Flash (in common mode).

Reply:
Why can't it be moved to the soc Kconfig file like I suggested? (the one without the `.soc` suffix). You should then be able to select it from any driver that needs this.

Reply:
Okay. (I missed that the file you names is without .soc)